src/service/webview.service.ts

```diff
@@ -353,19 +354,18 @@ export class WebviewService {
           vscode.commands.executeCommand(
             "vscode.open",
             vscode.Uri.parse(
-              `https://${
-                host.hostname
-              }/login/oauth/authorize?scope=gist%20read:user&client_id=cfd96460d8b110e2351b&redirect_uri=http://localhost:54321/callback`
+              `https://${host.hostname}/login/oauth/authorize?scope=gist%20read:user&client_id=cfd96460d8b110e2351b&redirect_uri=http://localhost:54321/callback`
             )
           );
           break;
+        }
         case "editConfiguration":
           this.OpenSettingsPage(
             await state.commons.GetCustomSettings(),
-            await state.commons.GetSettings()
+            state.commons.GetSettings()
```

Comment:
`await` is removed?

Reply:
Yes, it's actually not a Promise https://github.com/shanalikhan/code-settings-sync/blob/cfe6938691358a7f3fb7ee40baeef2f6d613f426/src/commons.ts#L371